### PR TITLE
EXT_clip_cull_distance: add GL_OES_shader_io_blocks interaction

### PR DESCRIPTION
**Pull request by zmike**

### CODE REVIEW

File: extensions/EXT/EXT_clip_cull_distance.txt [hunk 1]
```diff
@@ -273,6 +275,8 @@ Additions to Chapter 7 of the OpenGL ES Shading Language 3.20 Specification
         highp float gl_CullDistance[];
 
     };
+    [[ If OES_shader_io_blocks is supported: ]]
+    If I/O blocks are not in use, these variables are not in gl_PerVertex.
```
Comment (gnl21) [inline extensions/EXT/EXT_clip_cull_distance.txt:279]: I think this needs to apply when OES_shader_io_blocks is not supported as well.

Comment (zmike) [reply extensions/EXT/EXT_clip_cull_distance.txt:279]: So drop the `[[ ]]` conditional?

Comment (gnl21) [reply extensions/EXT/EXT_clip_cull_distance.txt:279]: Yes, I think so.

Comment (zmike) [reply extensions/EXT/EXT_clip_cull_distance.txt]: Done

**Review by gnl21 — APPROVED** [review]

**Comment by zmike** [comment]
@oddhack WG has approved, please merge.

**Pull request merged**